config.yaml

```diff
@@ -1,15 +1,24 @@
+# Refer to https://payu.readthedocs.io/en/latest/config.html for full details of the 
+# payu configuration settings.
+
+# PBS configuration
+
+# If submitting to a different project to your default, uncomment line below
+# and replace PROJECT_CODE with appropriate code. This may require setting shortpath
+# project: PROJECT_CODE
+
+# Force payu to always find, and save, files in this scratch project directory
+# shortpath: /scratch/PROJECT_CODE
+
+# Note: if laboratory is relative path, it is relative to shortpath/$USER
+laboratory: access-esm
+
 jobname: pre-industrial
 queue: normal
-walltime: 2:00:00
+walltime: 2:30:00
 
-modules:
-  use:
-      - /g/data/vk83/apps/spack/0.22/release/modules/linux-rocky8-x86_64_v4
-  load:
-      - access-esm1p5/2024.05.0
 
-# note: if laboratory is relative path, it is relative to /scratch/$PROJECT/$USER
-laboratory: access-esm
```

Comment:
Do we want to remove this? 

It used to be handy to reduce the possibility of "namespace collision" between other `access` models, but this is no longer an issue with the experiment ID changes, so perhaps it is ok.

Reply:
I've kept the laboratory setting in a bit higher up, but just had updated to comment to be accurate now that the `shortpath` setting has been added in:
```
# Note: if laboratory is relative path, it is relative to shortpath/$USER
laboratory: access-esm
```